src/stories/material-header/material-header.tsx

```diff
@@ -0,0 +1,82 @@
+import React from "react";
+import { AvailabilityLabel } from "../availability-label/AvailabilityLabel";
+import { ButtonFavourite } from "../button-favourite/ButtonFavourite";
+import { Button } from "../button/Button";
+import { Material } from "../material/Material";
+import MaterialHeaderText from "./material-header-text";
+import MaterialPeriodikumSelect from "./material-periodikum-select";
+
+interface MaterialHeaderProps {
+  title: string;
+  author: string;
+  periodikum?: boolean;
+  ctaText?: string;
+}
+
+const MaterialHeader: React.FC<MaterialHeaderProps> = ({
+  title,
+  author,
+  periodikum,
+  ctaText,
+}) => {
+  return (
+    <header className="material-header">
+      <div className="material-header__cover-container">
+        <Material
+          url="images/book_cover_3.jpg"
+          size="large"
+          tint="120"
+          animate={true}
+        />
+      </div>
+      <div className="material-header__content">
+        <ButtonFavourite />
+        <MaterialHeaderText title={title} author={author} />
+        <div className="material-header__availability-label-container">
+          <AvailabilityLabel
```

Comment:
I guess we could just map through an array of four values like they do it in Checkbox.stories.tsx
Check that one out ;)